modules/core/src/v2/coins/algo.ts

```diff
@@ -79,22 +77,31 @@ export interface HalfSignedTransaction {
   };
 }
 
+export interface TransactionFee {
+  fee: string;
+}
 export interface ExplainTransactionOptions {
   txHex?: string;
   halfSigned?: {
     txHex: string;
   };
+  publicKeys?: string[];
+  feeInfo: TransactionFee;
 }
 
 export interface VerifiedTransactionParameters {
   txHex: string;
   addressVersion: number;
-  keys: string[];
-  sk: string;
+  signers: string[];
+  prv: string;
   isHalfSigned: boolean;
+  numberSigners: number;
 }
 
 export class Algo extends BaseCoin {
+  readonly ENABLE_TOKEN = 'enabletoken';
```

Comment:
nit: these seem out of place here - are these types of transfers? If so they should be declared as `export type TransferType = 'enabletoken' | 'disabletoken';` so we can use the compiler to correctly enforce the usage of these string constants.